custom/abdm/milestone_one/utils/response_util.py

```diff
@@ -8,6 +8,11 @@
 success_response_keys = ["txnId", "healthIdNumber", "auth_methods", "token",
                          "health_card", "health_id", "authMethods"]
 
+ABHA_IN_USE_ERROR_CODE = 1001
+ERROR_CODES_DETAIL = {
```

Comment:
this could be `ERROR_MESSAGES`?

Reply:
additionally, could this in a const file? I am not sure if reponse_util is the best module to keep this in.

Reply:
Sounds great!  Addressed [here](https://github.com/dimagi/commcare-hq/pull/33006/commits/0eb179134c466d45c31eff38e6ce04609c8a1c41)

Reply:
Cool